packages/iris-grid/src/IrisGridModel.ts

```diff
@@ -321,6 +321,13 @@ abstract class IrisGridModel<
     return EMPTY_ARRAY;
   }
 
+  /**
+   * @returns Names of key columns
+   */
+  get keyColumnSet(): Set<ColumnName> {
+    return new Set();
```

Comment:
We don't want to create a new set each time this property is accessed. Since it's just an empty set, you can declare an empty set as a constant at the top (similar to `EMPTY_ARRAY` which we use in other locations; would be `const EMPTY_SET: Set<never> = new Set();`